Frontend/src/Pages/Authentication/SignUp.jsx

```diff
@@ -1,9 +1,119 @@
-import React from 'react'
+import React from 'react';
+import TextInput from '../../components/utility/textInput';
+import Button from '../../components/utility/button';
+import ThirdPartyButton from '../../components/utility/thirdPartyButton';
+import Image from '../../assets/images/Frame 422.png';
+import { FcGoogle } from "react-icons/fc";
+import { ImGithub } from "react-icons/im";
+import { useFormik } from 'formik';
+import * as Yup from 'yup';
+
+
+const SignUpSchema = Yup.object().shape({
+  name: Yup.string().min(2, 'Name is too Short').max(50, 'Name is too Long!').required('Required'),
+  email: Yup.string().email('Invalid email').required('Required'),
+  password: Yup.string().required('Required'),
```

Comment:
Please include restriction to password strength such as 
- at least 8 Characters or Longer,
- include unique characters etc..